src/formio/components/Fieldset.stories.mdx

```diff
@@ -0,0 +1,146 @@
+import {Canvas, Meta, Story, ArgsTable} from '@storybook/addon-docs';
+import {FormioComponent, FormDecorator} from './story-util';
+
+<Meta
+  title="Form.io components/Fieldset"
+  argTypes={{
+    components: {
+      type: {name: 'array'},
+    },
+    id: {
+      type: {name: 'string'},
+    },
+    label: {
+      type: {name: 'string', required: false},
+    },
+    descriptionFamilyName: {
+      type: {name: 'string'},
+    },
+    descriptionGivenName: {
+      type: {name: 'string'},
+    },
+    labelFirstName: {
+      type: {name: 'string'},
+    },
+    labelLastName: {
+      type: {name: 'string'},
+    },
+    labelPrivacy: {
+      type: {name: 'string'},
+    },
+    placeholderFirstName: {
+      type: {name: 'string'},
+    },
+    placeholderLastName: {
+      type: {name: 'string'},
+    },
+  }}
+  decorators={[FormDecorator]}
+  parameters={{
+    controls: {sort: 'requiredFirst'},
+  }}
+/>
+
+# Field Set
+
+A Field Set can be used to create a title of an area of the form. This is useful to put inside
+Layout components or in between lots of related components. The Field Set is for display only and
+will not be saved to the API,
```

Comment:
not entirely correct?

A fieldset is used to group a number of fields. The provided label will be used as title of the fieldset. Optionally, you can [hide](https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/form/fieldset.js#L18) this title.

A fieldset is a presentation-only component - no data is saved to the API.